replication-commands/src/main/java/org/codice/ditto/replication/commands/ConfigDeleteCommand.java

```diff
@@ -91,6 +115,41 @@
   private static final int DEFAULT_START_INDEX = 0;
 
   @Override
+  protected final Object executeWithSubject() throws Exception {
+    if (configNames == null || configNames.isEmpty()) {
+      if (deleteMetacards) {
+        removeAllReplicatedMetacards();
+        removeReplicationItems(null);
+      }
+
+      if (deleteHistory) {
+        removeHistoryItems(null);
+      }
+
+      if (deleteConfig) {
+        replicatorConfigLoader
```

Comment:
❓  Since the above two cases are handled by private methods, wouldn't this one makes it consistent to also be done in a separate private method?